### ISSUE TITLE
URI bundle schemes

### ISSUE THREAD

**Issue by jasonsturges** [jamsyoung/apple-news-format-snippets#1]
Your README states URI ANF type must begin with `http://` or `https://`; however, `bundle://` scheme is also supported for local assets.

For example, from the [`Image`](https://developer.apple.com/documentation/apple_news/image) component documentation:

> **URL** **Required** The URL of an image file. 
> `string` Image URLs can begin with http://, https://, or bundle://. If the image URL begins with bundle://, the referenced image file must be in the same directory as the document.
>
> ```
> {
>   "role": "image",
>   "URL": "bundle://summer.jpg",
>   "caption": "Thanks to the record drought, mountain lions have begun to descend from the peaks, sometimes into urban settings."
> }
> ```

Cool project, thanks for putting this together.

**Comment by jamsyoung** [comment]
Good catch! Admittedly I never use bundle:// anymore myself and had forgotten about it!  I'll get that updated on the next update. Thanks!
